src/tests/tests_ref.spec.js

```diff
@@ -46,4 +49,39 @@ test.describe('Tests Unit 10', () => {
             await app.inventory.sortInventoriesTest(data.sortBy, productNames, productPrices);
         });
     });
+
+    // test adding to the cart 2 random product from inventory page
+    test('Adding several random products to the cart', async (
+    /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        // Adding 2 random products to the cart
+        const products = await app.inventory.addRandomProductsToCart();
+
+        // Verify count of product in the cart  (on the cart badge at the header)
+        test.info().annotations.push({
```

Comment:
The result of the report is 
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/e27cd5d7-5c46-4a91-b21f-778709ffb3ec">

I suppose you wanted to have something like this
<img width="1012" alt="image" src="https://github.com/user-attachments/assets/76b0ff35-2fbc-4053-b1ab-4744e9322e11">

In this case you have to use https://playwright.dev/docs/api/class-teststep